apps/poker-roll/src/app/queries/helpers.js

```diff
@@ -0,0 +1,86 @@
+import { useQuery, useMutation, useQueryClient } from '@tanstack/react-query';
+
+
+export function useUser() {
+  return useQuery({ queryKey: ['user'], queryFn: () => {
+    return fetch('http://localhost:3333/user').then( (res) => res.json(),);
+  } });
+}
+
+export function useSessions() {
+  return useQuery({ queryKey: ['sessions'], queryFn: () => {
+    return fetch('http://localhost:3333/user/session').then( (res) => res.json(),)
```

Comment:
Missing the `/api` prefix on these two endpoints.

Reply:
Done